src/mailcheck.js

```diff
@@ -28,10 +28,49 @@ var Mailcheck = {
 
   defaultSecondLevelDomains: ["yahoo", "hotmail", "mail", "live", "outlook", "gmx"],
 
-  defaultTopLevelDomains: ["com", "com.au", "com.tw", "ca", "co.nz", "co.uk", "de",
-    "fr", "it", "ru", "net", "org", "edu", "gov", "jp", "nl", "kr", "se", "eu",
-    "ie", "co.il", "us", "at", "be", "dk", "hk", "es", "gr", "ch", "no", "cz",
-    "in", "net", "net.au", "info", "biz", "mil", "co.jp", "sg", "hu", "uk"],
+  defaultTopLevelDomains: [
+    "at",
+    "be",
+    "biz",
+    "ca",
+    "ch",
+    "co.il",
+    "co.jp",
+    "co.nz",
+    "co.uk",
+    "com.au",
+    "com.tw",
+    "com",
+    "cz",
+    "de",
+    "dk",
+    "edu",
+    "es",
+    "eu",
+    "fr",
+    "gov",
+    "gr",
+    "hk",
+    "hu",
+    "ie",
+    "in",
+    "info",
+    "it",
+    "jp",
+    "kr",
+    "mil",
+    "net.au",
+    "net",
```

Comment:
Such formatting of array makes catching bugs easier IMO (also code review process is easier). In the previous versions of the array `net` domain were defined twice and it was hard to spot it.